### PR TITLE
Use swagger-client instead of a custom implementation

### PR DESCRIPTION
**Pull request by davidyuk**
Breaking changes:
- drops `axiosConfig` support because it uses `cross-fetch` that should be lighter than `axios`

Future work:
- write some tests for swagger.js
- investigate it is possible to initialize swagger.js in sync by a prefetched spec file
- generate endpoint handlers on build time to avoid extra request to the node

### CODE REVIEW

File: es/utils/swagger.js [hunk 1]
```diff
@@ -16,513 +16,82 @@
...
+ * @return {Object} Swagger client
+ * @example (await genSwaggerClient('https://mainnet.aeternity.io/api')).getAccountByPubkey('ak_jupBUgZNbcC4krDLR3tAkw1iBZoBbkNeShAq4atBtpFWmz36r')
+ */
+export default async (specUrl, internalUrl) => {
+  const spec = await (await fetch(specUrl)).json();
+
+  ['claim', 'preclaim', 'transfer', 'revoke', 'update'].forEach(name => {
+    const s = spec.paths[`/debug/names/${name}`]?.post
+    if (s && !s.consumes) s.consumes = ['application/json']
+  })
```
Comment (davidyuk) [inline es/utils/swagger.js:48]: this should be removed after resolving https://github.com/aeternity/aeternity/issues/3539

Comment (mradkov) [reply es/utils/swagger.js:48]: Can you log this into an issue, or mark this PR as `wip` until that is fixed?

Comment (davidyuk) [reply es/utils/swagger.js:48]: I have created #1124

**Review by mradkov — APPROVED** [review]
LGTM

**Pull request merged**